### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,47 +32,47 @@ The following table provides a detailed list of all models materialized within t
...
+- mara
+- pa0000
+- pa0001
+- pa0007
+- pa0008
+- pa0031
+- ska1
+- t001
+- t503
+- t880
```
Comment (fivetran-joemarkiewicz) [inline README.md:64]:
```suggestion
   - bkpf
   - bseg
   - faglflexa
   - faglflext
   - kna1
   - lfa1
   - mara
   - pa0000
   - pa0001
   - pa0007
   - pa0008
   - pa0031
   - ska1
   - t001
   - t503
   - t880
```

File: README.md [hunk 2]
```diff
@@ -32,47 +32,47 @@ The following table provides a detailed list of all models materialized within t
...
+## How do I use the dbt package?
+### Step 1: Prerequisites
 To use this dbt package, you must have the following:
 - At least one Fivetran of the following SAP connectors:
-   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
-   - [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
-   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 
+- [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
+- [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
+- [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana)
```
Comment (fivetran-joemarkiewicz) [inline README.md:47]:
```suggestion
   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
   - [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana)
```

File: README.md [hunk 3]
```diff
@@ -32,47 +32,47 @@ The following table provides a detailed list of all models materialized within t
...
 | [sap__0gl_account_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0gl_account_attr)    |   This model is used for loading G/L Account Number master data, originating from the ska1 source.                 |
 | [sap__0material_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0material_attr)    |  This model is used to display material attribute information, originating from the mara source.                                                           |
 | [sap__0vendor_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0vendor_attr)    |  This model is used to display vendor attributes, originating from the lfa1 source.                           |
 | [sap__0vendor_text](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0vendor_text) |  This model is used to display vendor text, originating from the lfa1 source.     |
 
-# 🎯 How do I use the dbt package?
-## Step 1: Prerequisites
+## How do I use the dbt package?
```
Comment (fivetran-joemarkiewicz) [inline README.md:41]:
```suggestion
<!--section-end-->

## How do I use the dbt package?
```

File: README.md [hunk 4]
```diff
@@ -12,16 +12,16 @@
...
 - Generates a comprehensive data dictionary of your source and modeled sap data through the [dbt docs site](https://fivetran.github.io/dbt_sap/).
 
 
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.
 > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_sap/#!/overview?g_v=1).
 
 | **model**                         | **description**                                                                                                                                                                                                                             |
```
Comment (fivetran-joemarkiewicz) [inline README.md:27]:
```suggestion
<!--section=“sap_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_sap/#!/overview?g_v=1).

| **Table**                         | **Description**                                                                                                                                                                                                                             |
```

**Comment by fivetran-joemarkiewicz** [comment]
I'm aware the integration tests aren't passing. However, I believe this is an issue with the underlying code and should be addressed in a separate PR.

**Review by fivetran-catfritz — APPROVED** [review]
lgtm

**Pull request merged**